bpytop.py

```diff
@@ -4979,6 +4993,12 @@ def process_keys():
 				Collector.collect(ProcCollector, interrupt=True, redraw=True)
 			elif key == "f":
 				ProcBox.filtering = True
+				ProcCollector.case_sensitive = False
+				if not ProcCollector.search_filter: ProcBox.start = 0
+				Collector.collect(ProcCollector, redraw=True, only_draw=True)
+			elif key == "F":
+				ProcBox.filtering = True
+				ProcCollector.case_sensitive = True
```

Comment:
This could be a bit more efficient like:
```python
elif key in ["f", "F"]:
				ProcBox.filtering = True
				ProcCollector.case_sensitive = key == "F"
				if not ProcCollector.search_filter: ProcBox.start = 0
				Collector.collect(ProcCollector, redraw=True, only_draw=True)
´´´